fix-nix-credentials.sh

```diff
@@ -34,7 +34,7 @@ mkdir -p "$HOME/.config/nix";
 echo "machine api.github.com password $INPUTS_GITHUB_ACCESS_TOKEN" >> "$HOME/.netrc";
 echo "machine pkgs.github.com password $INPUTS_GITHUB_ACCESS_TOKEN" >> "$HOME/.netrc";
 echo "machine github.com password $INPUTS_GITHUB_ACCESS_TOKEN" >> "$HOME/.netrc";
-ln -sr "$HOME/.netrc" "$HOME/.config/nix/netrc";
+ln -s "$HOME/.netrc" "$HOME/.config/nix/netrc";
```

Comment:
@aakropotkin was there a need for a relative symlink for some reason?

Reply:
No they're equivalent here.